### ISSUE TITLE
DNN

### ISSUE THREAD

**Issue by jikan1221** [jikan1221/JK_BLOG#15]
logical regression-->>$$\sigma(W^T , B) = \mathbb{P}(Y = 1 | X = X^* )=\frac{1}{1 + exp(\beta^0 X_0 + \beta^1 X_1 + ...+\beta^N X_N + \sigma )} $$.
 $$Loss function =-( \lg Y )* Y^{x}(W,B) + (1 - Y) * \lg {(1 -Y^x(W,B) )}$$
$$if ,Y^{x}(W,B) \approx Y, Loss prone to Mix $$
if it is Matrix, $$softMax = [ \frac {e^{Y_i}}{e^{Y_i + ...}}], \sum_1^n Soft = 1$$
##N-row, K-feature 
$$X_{(N, K)} \approx A_{(K * P)} \approx X_{(P,M)} \approx B_{(M, J)} \approx X_{J,K} \approx  \mathbb{I}_{K, 1} , I_2 \in (0,1)$$
the result I will be a N-row,second col is 0 otherwise 1;
